lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Accordion.style';
+import type { AccordionHeaderProps, AccordionPanelProps, AccordionProps } from './types';
+
+const AccordionHeader = ({
+  children,
+  className,
+  ariaLabel,
+  accordionPanelId,
+}: AccordionHeaderProps): Node => {
+  const [isExpanded, toggleExpanded] = useState(false);
+
+  const handleClick = () => toggleExpanded(!isExpanded);
+
+  const handleOnKeyPress = event => event.keyCode === 13 && toggleExpanded(!isExpanded);
+
+  return (
+    <summary
+      className={className}
+      role="button"
+      aria-expanded={isExpanded}
+      aria-label={ariaLabel}
+      aria-labelledby={accordionPanelId}
+      onClick={handleClick}
+      onKeyPress={handleOnKeyPress}
+      tabIndex="0"
```

Comment:
You can't add tabIndex="0" by default, it will bring default focus from the page to this element. If we need to make it focusable we should conditionally set and remove it tabIndex="-1"

Reply:
As discussed, I validated tab indexing on accordion after adding button before it. It works fine and the tab index comes on button before accordion. Hence, keeping the change as it is.